src/commands/reset.js

```diff
@@ -46,6 +46,7 @@ class ResetCommand extends BaseCommand {
             {
               title: 'Cleanup Core data dir',
```

Comment:
Since we store data in volumes we don't need to remove dirs anymore and everything will be done by "Remove Docker containers and associated data" task